.travis/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+set -x
+
+ci_requirements="pip setuptools tox virtualenv"
+
+if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+    if [[ ${TOXENV} == *"py27"* ]]; then
+        # install pip on the system python
+        curl -O https://bootstrap.pypa.io/get-pip.py
+        python get-pip.py --user
+        python -m pip install --user virtualenv
+        python -m virtualenv .venv/
```

Comment:
I don't see that we need a custom virtualenv. tox will automatically create one for us.

Reply:
Removed